qqqwww123cdv/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def data
+  file = File.new('data.txt', 'r:UTF-8')
+  file.read
+end
+
+def weather_data(town, data)
+  data.split("\n").find do |value|
+    if value.include? town
```

Comment:
please take a look at `#find` usage here https://apidock.com/ruby/Enumerable/find
there are a lot of redundant operations